packages/lib-classifier/src/store/ClassificationStore.js

```diff
@@ -42,6 +42,24 @@ const ClassificationStore = types
   .actions(self => {
     function afterAttach () {
       createSubjectObserver()
+
+      onAction(self, (call) => {
+        const feedback = getRoot(self).feedback
+        switch (call.name) {
+          case 'addAnnotation': {
```

Comment:
We don't really need to compute feedback each time an annotation changes. We only really need to complete feedback when a classification is complete, at which point we can iterate through the annotations and call `feedback.update` on each one.